AWS/legos/aws_filter_ebs_volumes_with_low_iops/aws_filter_ebs_volumes_with_low_iops.py

```diff
@@ -0,0 +1,65 @@
+##
+##  Copyright (c) 2023 unSkript, Inc
+##  All rights reserved.
+##
+from pydantic import BaseModel, Field
+from typing import Optional, Tuple
+from unskript.legos.aws.aws_list_all_regions.aws_list_all_regions import aws_list_all_regions
+import pprint
+
+
+class InputSchema(BaseModel):
+    region: Optional[str] = Field(
+        default="",
+        title='Region',
+        description='AWS Region.')
+    iops_threshold: Optional[int] = Field(

```

Comment:
si region and iops really optional?

If they are not included will the Lego run?

If you convert Optional[str] to str - they become required.

Reply:
Maybe in the readme say - if not provided - the value is set to 100

Reply:
I will add this point in the description and yes if both inputs are not provided then also Lego will run.